spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Threshold.scala

```diff
@@ -34,10 +34,9 @@ import scala.reflect.ClassTag
  * @param ip inplace mode
  */
 
-@SerialVersionUID(3953292249027271493L)
```

Comment:
Keep the UID in old layers unchanged. Others may propose a PR to remove all of them.
For new layers, we simply don't add UID.